### PR TITLE
Misc Tidy up

### LINKED ISSUE #41: section numbering and file names
Current section files have multiple sections inside each which means that they cannot be numbered sequentially.

### LINKED ISSUE #62: David Stansby
## Details

* Given Name: David

* Family Name: Stansby

* Email: dstansby@gmail.com

* Affiliation: Imperial College London

* ORCID (if available): 0000-0002-1365-1908

## Contribution to the SunPy Community

Reporting and fixing bugs, and providing several improvements to code testing.

### LINKED ISSUE #65: Hopping on as a co-author
## Details

* James Paul

* Mason

* james.p.mason@nasa.gov

* NASA Goddard Space Flight Center 

* ORCID: 0000-0002-3783-5509

## Contribution to the SunPy Community
Updated some of the code comments related to SDO/EVE instrument and patched up some time functions: https://github.com/sunpy/sunpy/commits?author=jmason86

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes #41
Fixes #62
Fixes #65

Addressed the minor issues from #63

So I decided in my infinite wisdom to just shorten some of the affiliations and expanded upon some of them.
Changed numpy to NumPy (all I could find was one), GSOC to GSoC (again once), open-source to open source (a few times). 
Removed the inline bullet points for Map and TimeSeries.

### CODE REVIEW

File: sections/13-acknowledgements.tex [hunk 1]
```diff
@@ -0,0 +1,14 @@
+\acknowledgments
+
+We would like to acknowledge financial contributions by Google as part of the Google Summer of Code program.
+This work has made use of data from the European Space Agency (ESA) mission {\it Gaia} (\url{https://www.cosmos.esa.int/gaia}), processed by the {\it Gaia} Data Processing and Analysis Consortium (DPAC, \url{https://www.cosmos.esa.int/web/gaia/dpac/consortium}).
```
Comment (bsipocz) [inline sections/13-acknowledgements.tex:4]: weird to acknowledge gaia so prominently just as it was used for one of the plot. Maybe a restructuring and rephrasing ("this paper has made use of data")?

Comment (nabobalis) [reply sections/13-acknowledgements.tex:4]: Yeah this was a merge of the older acknowledgments and the separate file you created.  I wasn't sure what to do with the old one.

Comment (bsipocz) [reply sections/13-acknowledgements.tex:4]: oh, gotcha. I didn't see the old one 😅

Comment (nabobalis) [reply sections/13-acknowledgements.tex:4]: I didn't until I was searching for sections.

**Comment by ehsteve** [comment]
Thanks @nabobalis!

**Comment by nabobalis** [comment]
I was planning to do the table changes but I will open a new PR for that .

**Pull request merged**